### PR TITLE
initial overstroomik helm chart

### PR DESCRIPTION
**Pull request by avgils**

### CODE REVIEW

File: helm/overstroomik/values.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Default values for overstroomik.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+images:
+  geoserver: deltares/overstroomik-geoserver
+  backend: deltares/overstroomik-backend
+  pullPolicy: IfNotPresent
```
Comment (evetion) [inline helm/overstroomik/values.yaml:10]: Maybe set to Always?

File: helm/overstroomik/templates/backend-service.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+spec:
+  type: {{ .Values.backend.service.type }}
+  ports:
+    - port: {{ .Values.backend.service.port }}
+      targetPort: 80
+      protocol: TCP
+  selector:
+    app: {{ .Values.backend.name }}
```
Comment (evetion) [inline helm/overstroomik/templates/backend-service.yaml:14]: EOF

File: helm/overstroomik/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{{- if .Values.ingress.enabled -}}
```
Comment (evetion) [inline helm/overstroomik/templates/ingress.yaml:1]: Can you comment on how to generate these files in a README or similar? I hope that at least some of these templated .yaml files can be generated automatically.

**Review by evetion — CHANGES_REQUESTED** [review]
Small README addition on how to actually install/update/upgrade/delete the helm chart in your kubernetes cluster.

File: helm/overstroomik/templates/ingress.yaml (hunk 3 above)
Comment (avgils) [reply helm/overstroomik/templates/ingress.yaml:1]: helm create

File: helm/overstroomik/values.yaml (hunk 1 above)
Comment (avgils) [reply helm/overstroomik/values.yaml:10]: Why? I propose to make a tag in github build containers in hub from the tagged versions and specify the version of containers in helm file.

File: geoserver/Dockerfile [hunk 4]
```diff
@@ -1 +1,3 @@
 FROM oscarfonts/geoserver:2.18.0
+
+COPY data_dir/ /var/local/geoserver
```
[the remark this answers is not in the document]
Comment (avgils) [reply geoserver/Dockerfile:3]: :)

**Review by evetion — APPROVED** [review]

**Pull request merged**